src/kv/RocksDBStore.cc

```diff
@@ -181,6 +181,7 @@ int RocksDBStore::set_merge_operator(
 class CephRocksdbLogger : public rocksdb::Logger {
   CephContext *cct;
 public:
+  static thread_local int32_t left_to_print;
```

Comment:
Why not using atomic instead? IMO this implementation is a bit questionable - no guarantee that RocksDB logs (or will always log) everything from the same thread on startup